erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py

```diff
@@ -1,66 +1,72 @@
 # Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
```

Comment:
Why are there so many changes?! The formatting changes are making it impossible to see the actual changes. Either don't format, or move formatting to a separate commit.

Reply:
There are a few other changes that I had to do to avoid a bunch of duplication.

For the formatting, I can only suggest using [no whitespace](https://github.com/ParsimonyGit/erpnext/pull/5/files?w=1) 😅